### PR TITLE
[add] get detail of archive

### PR DESCRIPTION
**Pull request by fewdan**
### What problem does this PR solve?

get detail of archive

### What is changed and how does it work?
get detail of archive

### Check List

Tests

- Unit test

Code changes

- Has Go code change

Side effects

- NO

Related changes

- Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/apiserver/archive/experiment.go [hunk 1]
```diff
@@ -55,6 +56,8 @@ func Register(r *gin.RouterGroup, s *Service) {
 
 	// TODO: add more api handlers
 	endpoint.GET("", s.listExperiments)
+	endpoint.GET("/detailSearch", s.experimentDetailSearch)
```
Comment (yeya24) [inline pkg/apiserver/archive/experiment.go:59]: Can we use a better endpoint name?

Comment (fewdan) [reply pkg/apiserver/archive/experiment.go:59]: I have no idea. Any better suggestions about the name?

Comment (zhouqiang-cl) [reply pkg/apiserver/archive/experiment.go:59]: How about /detail/search?

File: pkg/store/experiment/experiment.go [hunk 2]
```diff
@@ -72,6 +73,57 @@ func (e *experimentStore) ListMeta(_ context.Context, kind, ns, name string) ([]
...
+	for _, archive := range archives {
+		if st.Before(archive.StartTime) {
+			st = archive.StartTime
+			UID = archive.UID
+		}
+	}
+	return UID, nil
+}
+
+func (e *experimentStore) DetailList(ctx context.Context, kind, namespace, name, uid string) ([]*core.ArchiveExperiment, error) {
```
Comment (zhouqiang-cl) [inline pkg/store/experiment/experiment.go:101]: Add comments?

File: pkg/apiserver/archive/experiment.go [hunk 3]
```diff
@@ -81,3 +84,68 @@ func (s *Service) listExperiments(c *gin.Context) {
...
+// @Summary Get the details of chaos experiment.
+// @Description Get the details of chaos experiment.
+// @Tags archives
+// @Produce json
+// @Param namespace query string false "namespace"
+// @Param name query string false "name"
+// @Param kind query string false "kind" Enums(PodChaos, IoChaos, NetworkChaos, TimeChaos, KernelChaos, StressChaos)
+// @Param uid query string false "uid"
+// @Success 200 {array} core.ArchiveExperiment
+// @Router /api/archives/detailSearch [get]
```
Comment (cwen0) [inline pkg/apiserver/archive/experiment.go:97]: detail/search?

**Pull request merged**